src/Sinch/Conversation/Messages/Send/SendMessageRequest.cs

```diff
@@ -154,11 +157,34 @@ public override string ToString()
             sb.Append("  ConversationMetadata: ").Append(ConversationMetadata).Append("\n");
             sb.Append("  Queue: ").Append(Queue).Append("\n");
             sb.Append("  Recipient: ").Append(Recipient).Append("\n");
-            sb.Append("  Ttl: ").Append(Ttl).Append("\n");
+            sb.Append("  Ttl: ").Append(TtlSeconds).Append("\n");
             sb.Append("  ProcessingStrategy: ").Append(ProcessingStrategy).Append("\n");
             sb.Append("  CorrelationId: ").Append(CorrelationId).Append("\n");
             sb.Append("}\n");
             return sb.ToString();
         }
     }
+
+    public class TimeToLiveConverter : JsonConverter<int?>
+    {
+        public override int? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            var str = reader.GetString();
+            if (str is null) return null;
+            str = str.Trim();
+            var val = str.TakeWhile(x => x == 's').ToString();
```

Comment:
This condition seems wrong: if the value is `10s`, it will return an empty string as the condition will fail when reading the first character.
Cf doc: https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.takewhile?view=net-8.0 => "Returns elements from a sequence as long as a specified condition is true, and then skips the remaining elements."

Reply:
ah yeah good catch, in test there is no reads of values as well as no API endpoint returns ttl anyware